app/models/induction_period.rb

```diff
@@ -32,6 +33,19 @@ def siblings
 
 private
 
+  # Ensure admin users inserting new induction periods include end dates.
+  def end_date_admin_only
+    return unless inserting_induction_period? && finished_on.blank?
+
+    errors.add(:finished_on, "End date is required for inserted periods")
+  end
+
+  def inserting_induction_period?
```

Comment:
`inserting_induction_period` is a bit confusing to me, probably because I am not as familiar with the domain.

How does `valid_for_insertion` sound to you?

Reply:
It is attempting to articulate "are you positioning this before, or between existing periods" i.e. are you inserting. No validation is happening at this point, it is the predicate that helps us know to validate.

If you are inserting, and have also forgotten to complete the field, then we error?

The option I went with first was just `#backfilling?` which is typically succinct from me but I thought it needed expanding. 

How about `#between_other_induction_periods?`